doc.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/services/models"
+	"github.com/kiali/kiali/status"
+)
+
+/////////////////////
+// SWAGGER PARAMETERS
+/////////////////////
+
+// A Namespace provide a scope for names.
+// This type used to describe a set of objects.
+//
+// swagger:parameters istioConfigList serviceValidations namespaceValidations objectValidations
+type NamespaceParam struct {
+	// The id of the namespace.
+	//
+	// in: path
+	// required: true
+	Name string `json:"namespace"`
+}
+
+// Service identify the a service object
+//
+// swagger:parameters serviceValidations
+type ServiceParam struct {
+	// The name of the service
+	//
+	// in: path
+	// required: true
+	Name string `json:"service"`
+}
+
+// Istio Object Type:
+//
+// swagger:parameters objectValidations
+type ObjectType struct {
+	// The type of the istio object
+	//
+	// in: path
+	// required: true
+	// pattern: ^(gateways|virtualservices|destinationrules|serviceentries|rules|quotaspecs|quotaspecbindings)$
+	Name string `json:"object_type"`
```

Comment:
Oh, we still have some snake-case json? I thought everything was moved to camel case :-(